qlib/linux_def.rs

```diff
@@ -2983,47 +2984,51 @@ impl MemoryDef {
     pub const PAGE_SIZE_2M_MASK: u64 = !(Self::PAGE_SIZE_2M - 1);
     pub const BLOCK_SIZE: u64 = 64 * Self::ONE_GB;
 
-    pub const PHY_LOWER_ADDR: u64 = 256 * Self::ONE_GB; // 256 ~ 512GB is Guest kernel space
-    pub const PHY_UPPER_ADDR: u64 = Self::PHY_LOWER_ADDR + 256 * Self::ONE_GB; // 256 ~ 512GB is Guest kernel space
+    const PHY_LOWER_ADDR: u64 = 256 * Self::ONE_GB; // 256 ~ 512GB is Guest kernel space
+    const PHY_UPPER_ADDR: u64 = Self::PHY_LOWER_ADDR + 256 * Self::ONE_GB; // 256 ~ 512GB is Guest kernel space
   
     pub const NVIDIA_START_ADDR: u64 = 0x200000000;
     pub const NVIDIA_ADDR_SIZE: u64 = 2 * Self::ONE_GB;
 
+    // heap
+    pub const HOST_INIT_HEAP_OFFSET: u64 = MemoryDef::NVIDIA_START_ADDR + Self::NVIDIA_ADDR_SIZE;
+    pub const HOST_INIT_HEAP_SIZE: u64 = 5 * Self::ONE_GB;
+    pub const HOST_INIT_HEAP_END: u64 = Self::HOST_INIT_HEAP_OFFSET + Self::HOST_INIT_HEAP_SIZE;
+
     // memory layout
     // PHY_LOWER_ADDR: qkernel image 512MB
     pub const QKERNEL_IMAGE_SIZE: u64 = 512 * Self::ONE_MB;
+
+    //==================================================== private ==============================================//
```

Comment:
Please don't use this LOOOOOONG separators. Limit the length to 72 or so, otherwise it will wrap very ugly.

Reply:
resolved